CHANGELOG.md

```
@@ -1,5 +1,35 @@
 # 更新日志
 
+2.1.0
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)